### PR TITLE
added aimToHub

### PR DESCRIPTION
**Pull request by yutytuty**

### CODE REVIEW

File: src/main/java/frc/robot/commands/autonomous/SuperAutonomous.java [hunk 1]
```diff
@@ -32,13 +36,25 @@ private DriveTankWithPID returnByEncoders() {
...
         return new DriveArcade(drivetrain, () -> 0.0, SEEK_ROTATE_VALUE).withInterrupt(() ->
-                -SEEK_HUB_TOLERANCE.get() <= limelight.getHorizontalOffsetFromTarget() &&
-                limelight.getHorizontalOffsetFromTarget() <= SEEK_HUB_TOLERANCE.get());
+                -SEEK_HUB_TOLERANCE.get() <= limelight.getHorizontalOffsetFromTargetInDegrees() &&
+                        limelight.getHorizontalOffsetFromTargetInDegrees() <= SEEK_HUB_TOLERANCE.get());
     }
 
+    private DriveArcadeWithPID aimToHub() {
+        Limelight limelight = drivetrain.getLimelight();
+        return new DriveArcadeWithPID(drivetrain, limelight::getHorizontalOffsetFromTargetInPixels, () -> 0.0, () -> AIM_TO_HUB_MOVE_VALUE,
```
Comment (tal-sitton) [inline src/main/java/frc/robot/commands/autonomous/SuperAutonomous.java:51]: but its wait time should be smaller than the cargoCam one

**Review by tal-sitton — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/autonomous/SuperAutonomous.java [hunk 2]
```diff
@@ -18,27 +20,38 @@
     private static final Supplier<Double> SEEK_ROTATE_VALUE = rootNamespace.addConstantDouble("seek rotate value", 0.5);
     private static final Supplier<Double> SEEK_HUB_TOLERANCE = rootNamespace.addConstantDouble("seek hub tolerance", 18);
 
+    private static final double AIM_TO_HUB_MOVE_VALUE = 0.55;
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/commands/autonomous/SuperAutonomous.java:23]: this is quite slow and we don;t have the time

File: src/main/java/frc/robot/commands/autonomous/SuperAutonomous.java [hunk 3]
```diff
@@ -18,27 +20,38 @@
...
-        return new DriveTankWithPID(drivetrain, drivetrain.getEncodersPIDSettings(), drivetrain.getEncodersPIDSettings(),
-                drivetrain.getLeftDistance() / 2, drivetrain.getRightDistance() / 2,
-                drivetrain::getLeftDistance, drivetrain::getRightDistance);
+    /**
+     * @return a command that turns the robot until the angle between the limelight and target is small enough
+     */
+    private Command seekHub() {
+        Limelight limelight = drivetrain.getLimelight();
+        return new DriveArcade(drivetrain, () -> 0.0, SEEK_ROTATE_VALUE, () ->
+                -SEEK_HUB_TOLERANCE.get() <= limelight.getHorizontalOffsetFromTargetInDegrees() &&
```
Comment (Yoel-Perman-Brilliant) [inline src/main/java/frc/robot/commands/autonomous/SuperAutonomous.java:42]: Math.abs

**Review by Yoel-Perman-Brilliant — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/autonomous/SuperAutonomous.java (hunk 2 above)
Comment (yutytuty) [reply src/main/java/frc/robot/commands/autonomous/SuperAutonomous.java:23]: This is temporary. I will change it when we calibrate.

File: src/main/java/frc/robot/commands/autonomous/SuperAutonomous.java [hunk 4]
```diff
@@ -18,27 +20,37 @@
...
-        return new DriveTankWithPID(drivetrain, drivetrain.getEncodersPIDSettings(), drivetrain.getEncodersPIDSettings(),
-                drivetrain.getLeftDistance() / 2, drivetrain.getRightDistance() / 2,
-                drivetrain::getLeftDistance, drivetrain::getRightDistance);
+    /**
+     * @return a command that turns the robot until the angle between the limelight and target is small enough
+     */
+    private Command seekHub() {
+        Limelight limelight = drivetrain.getLimelight();
+        return new DriveArcade(drivetrain, () -> 0.0, SEEK_ROTATE_VALUE, () ->
+                Math.abs(limelight.getHorizontalOffsetFromTargetInDegrees()) <= SEEK_HUB_TOLERANCE.get());
```
Comment (tal-sitton) [inline src/main/java/frc/robot/commands/autonomous/SuperAutonomous.java:42]: && the limelight has a target

**Review by tal-sitton — CHANGES_REQUESTED** [review]

**Pull request merged**